tests/variant_calling_muse.nf.test

```diff
@@ -52,8 +57,13 @@ nextflow_pipeline {
                     stable_name,
                     // All files with stable contents
                     stable_path,
+                    // MuSE txt files will differ in their first line (holds Build Date & Time) between conda and docker/singularity
+                    txt_files.collect{ file -> [ file.getName(), file.readLines()[2..-1] ] },
                     // All bam files
-                    bam_files.collect{ file -> [ file.getName(), bam(file.toString(), fasta).getReadsMD5() ] }
+                    bam_files.collect{ file -> [ file.getName(), bam(file.toString(), fasta).getReadsMD5() ] },
+                    // All vcf files
+                    vcf_files.collect { file -> [file.getName(), path(file.toString()).vcf.variantsMD5] },
+                    count_files.collect{ file -> [ file.getName(), file.readLines() ] },
```

Comment:
Now that we know that the contents are stable, how about we md5 it?
https://www.nf-test.com/docs/assertions/files/#md5-checksum